build.assets/tooling/lib/git/git.go

```diff
@@ -0,0 +1,34 @@
+package git
+
+import (
+	"bytes"
+	"os/exec"
+	"strings"
+
+	"github.com/gravitational/trace"
+)
+
+// gitIsAvailable returns status of git
+func GitIsAvailable() error {
```

Comment:
```suggestion
// IsAvailable returns status of git
func IsAvailable() error {
```

`git.GitIsAvailable` stutters.